simapp/Makefile

```
@@ -24,4 +24,4 @@ BUILD_FLAGS := -ldflags '$(ldflags)'
 build:
 	@go build -mod=readonly $(BUILD_FLAGS) -o $(PWD)/build/ ./...
 
-.PHONY: build
+.PHONY: build
```

Let's add back the newline here to minimize changes 😄

```suggestion
.PHONY: build

```